doc/book/getting-started/forms-and-actions.md

```diff
@@ -102,15 +102,6 @@ class Album implements InputFilterAwareInterface
         $this->title  = ! empty($data['title']) ? $data['title'] : null;
     }
 
-    public function getArrayCopy()
```

Comment:
I meant it is already in the code during the "Adding new albums phase" while it's is actually added during the "Editing an album" phase later on in the tutorial. If you don't remove this, the part where you add it later on should be removed.